### ISSUE TITLE
[BUG] pressing ALT should show hotkey letters

### ISSUE THREAD

**Issue by DirkHaar** [linuxmint/cinnamon#11044]
```
 * Cinnamon version 5.49
   - Please specify if you are using the daily builds PPA
     ---   no   ---
 * Distribution - Mint 21
 * Graphics hardware *and* driver used Intel HD Graphics 620, driver i915
 * 64 bit
 * Attach ~/.xsession-errors, or /var/log/syslog
 see attachments @ #11043 , not related/relevant imho 

### Issue
- window system menu shortcuts/hot keys don't work anymore since mutter rebase (minimize, maximize, move/ grow/shrink...),
- no underlined letters anymore, not even when ALT is pressed, not working neither when trying noentheless
- hot keys had been located (and were manually changeable via hex editor*) in muffin.mo, that of course does not work anymore, too
              (*was always necessary in the past since x and m in Gerrman version do not correlate to Windows,
                  which is annoying as f###, aka PITA, if you use both systems at the same time) 

### Steps to reproduce
open a standard program/nemo window, press ALT-Space, try "x" to maximize or "m" to minimize.


### Expected behaviour
pressing ALT should show hotkey letters (where has the setting switch gone?)
(it should be - however - possible to change those)


### Other information
na

### Configuration info, if applicable
na

**Comment by DirkHaar** [comment]
(The file "/usr/share/locale/de/LC_MESSAGES/muffin.mo" still exists but I neither find all the current menu text strings nor does a change - adding a _ before the preferred hot key letter as before in Mint 20.3, unsure which Cinnamon version - has any effect.)

**Comment by mtwebster** [comment]
Check out the 'mnemonic-5-4-window-menu@mtwebster' extension. It wasn't done in time to merge in Cinnamon directly.

**Comment by DirkHaar** [comment]
I love you:-)
And I appreciate the much easier way to manipulate the hotkeys (though I had a script with "sed" commands to do it).
I wish it worked for Firefox this way, too.

**Comment by DirkHaar** [comment]
Easy workaround, and as included in follow-up versions this bug can be closed.

**Comment by DirkHaar** [comment]
@mtwebster: Where can the shortcuts be found? In the German translation for 21.1 someone found another way to screw up the "correct" ones (for "m_inimize" it's 'Ver_kleinern"), but this string can't be found (as in older versions in muffin.mo), maybe the file where it's stored is compressed/compiled.

**Comment by mtwebster** [comment]
They're part of Cinnamon now:
https://github.com/linuxmint/Cinnamon/blob/master/js/ui/windowMenu.js#L188-L344

Unfortunately the way our translation 'host' (Launchpad) works, it doesn't retain old translations, so these were performed anew, maybe explaining the errors, though.

I *did* link to the old .po files so they could copy/paste if they wanted, but I when we extracted the msgids from our code, comments ended up not being included. I had linked to the old muffin .po files there.  That's fixed now: bf8fb4f426225a8.
